### PR TITLE
Sanskar1306 patch 1

### PR DESCRIPTION
**Pull request by sanskar1306**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6063"></a>

### CODE REVIEW

File: components/map/Map.js
Comment (kumarsonsoff3) [inline components/map/Map.js:1]: Profile adding PR should not contain changes in any other existing file. So Please **revert** the changes from `Map.js`. 🛠

If you want to make any other improvement into the project, then please have a look at the [Contributing Guide](https://github.com/EddieHubCommunity/LinkFree/blob/main/CONTRIBUTING.md) first. Create an issue and get assigned by the maintainer to work on that.

File: data/sanskar1306.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/sans13_k",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/sanskar-kabra-a3a347178",
+      "icon": "FaLinkedIn"
+    },
```
Comment (kumarsonsoff3) [inline data/sanskar1306.json:15]: You probably be using an invalid icon here. It should be `FaLinkedin`. Also, remove this trailing comma `,` here.

```suggestion
      "icon": "FaLinkedin"
    }
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Also, you'd merge conflicts with the file `Map.js` probably due to your outdated branch. Please update your fork branch to resolve conflicts.

**Review by sanskar1306 — COMMENTED** [review]
Thanks .

**Pull request closed without merging**